pkg/mempool/simplemempool/internal/parts/formbatchesext/formbatches.go

```diff
@@ -21,7 +21,7 @@ func IncludeBatchCreation(
 		return nil
 	})
 
-	mpdsl.UponRequestBatch(m, func(origin *mppbtypes.RequestBatchOrigin) error {
+	mpdsl.UponRequestBatch(m, func(_ tt.EpochNr, origin *mppbtypes.RequestBatchOrigin) error {
```

Comment:
Yes, because this "external" version of the mempool doesn't need it. It's just an adapter to some other mempool implementation. We use it to hook to the Lotus mempool. Our own mempool (`.../formbatchesint/formbatches.go`) does use it.